packages/cardhost/app/templates/components/cards/cardstack/base-card/isolated.hbs

```diff
@@ -3,9 +3,14 @@
 {{!-- TODO use a sanitized version of the @card.id in the class name after we have css coming from card documents --}}
 <div
   data-test-isolated-card={{@card.name}}
-  class="isolated-card cardstack_base-card"
+  class="isolated-card cardstack_base-card template_{{@templateId}} {{@cardId}} {{@mode}}"
```

Comment:
Same comment as above. Instead you should do this:
```hbs
class="isolated-card {{if @card.adoptedFrom @card.adoptedFrom.name "cardstack_case-card"}} {{@mode}}"
```

Reply:
Also, let's hoist this class to the card renderer so that the user cant break this--as the card template will eventually be edited by the user.